packages/meteor-accounts-ui-bootstrap-3/package.js

```diff
@@ -0,0 +1,89 @@
+Package.describe({
```

Comment:
I had to include this package to update its dependency from the Meteor stylus package to the coagmano:stylus package.